tests/test_dbAuth.py

```diff
@@ -420,14 +419,5 @@ def test_urlEncoding(self):
             "%2A%5B%5D@host.example.com:5432/my_database")
 
 
-class MemoryTester(lsst.utils.tests.MemoryTestCase):
```

Comment:
Yes. We weren't using it anywhere else though. I had been thinking of using pytest-openfiles instead but I've never been able to convince pytest-openfiles that there was a leak when I experimented with it in the past.

Reply:
I've added pytest-openfiles and it found a real problem: our "test repo" tests that were meant to be using in-memory registry for speed were in fact using sqlite files because we had the wrong argument for makeRepo. We should add it to the conda env.